### PR TITLE
refactor(documentation): generate docs from TSDoc and typescript

### PR DESCRIPTION
**Pull request by martinkaintas**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+@aeternity/aepp-sdk / [Exports](modules.md)
```
Comment (subhod-i) [inline docs/README.md]: Can we not copy the main readme to here? Duplicate of `../README.md`

**Comment by subhod-i** [comment]
@martinkaintas The goal is to have  `docs/api-reference.md` file that containers all the SDK exported classes and methods only. All methods are in a single file that way it will be easier for developers to search for methods. Although, we can keep the rest of the supporting files under `docs/api-reference/` directory

**Comment by martinkaintas** [comment]
> @martinkaintas The goal is to have `docs/api-reference.md` file that containers all the SDK exported classes and methods only. All methods are in a single file that way it will be easier for developers to search for methods. Although, we can keep the rest of the supporting files under `docs/api-reference/` directory

@subhod-i I don't think putting it all into one file is a possibility with this tool. Maybe we want to switch to the `.html` version which also provides a search bar? It would look like this
image

**Comment by subhod-i** [comment]
@martinkaintas This looks better than the markdown and I observed many formatting issues in the markdown.

Screenshot 2022-06-02 at 12 12 31 PM

As @davidyuk said, we may have to migrate to HTML, I hope the effort is trivial.

**Pull request closed without merging**